src/Uno.UWP/UI/Colors.cs

```diff
@@ -16,20 +16,9 @@ static partial class Colors
 	{
 		private static Dictionary<string, Color> _colorMap = new Dictionary<string, Color>(StringComparer.OrdinalIgnoreCase);
 
-		public static Color FromARGB(byte a, byte r, byte g, byte b)
-		{
-			return Color.FromArgb(a, r, g, b);
-		}
+		public static Color FromARGB(byte a, byte r, byte g, byte b) => new(a, r, g, b);
 
-		public static Color FromInteger(int color)
-		{
-			return FromARGB(
-				(byte)((color & 0xFF000000) >> 24),
-				(byte)((color & 0x00FF0000) >> 16),
-				(byte)((color & 0x0000FF00) >> 8),
-				(byte)((color & 0x000000FF))
-			);
-		}
+		public static Color FromInteger(int color) => new((uint)color);
```

Comment:
This change can cause regressions. Previously the produced `Color` would have correct values for `A`, `R`, `G`, and `B` properties. Now they will be zero.

Reply:
Ignore this comment. I just saw your comment above.